README.md

```diff
@@ -17,19 +17,49 @@ conda activate c3po-os-api
 
 7. Install dependencies and dev dependencies locally from the `Pipfile` by running `pipenv install --dev`.
 
-8. Start the app `pipenv run uvicorn app.api:versioned_app`.
-
-9. Post a request in a separate shell!
-```bash
-curl -X POST -H "Content-Type: application/json" \
-    -d '{"text": "A fun little note."}' \
-    http://localhost:8000/predict
+8.  Create a `secrets.env` file and ensure it is on the `.gitignore`.  Add the following for local dev:
```

Comment:
replace "Create a `secrets.env`" with "In c3po-model-server/app/core/env_var, create a `secrets.env`"

Reply:
Fixed